extensions/PhabBugz/lib/Feed.pm

```diff
@@ -293,7 +293,7 @@ sub process_revision_change {
     $phab_users = get_phab_bmo_ids({ phids => \@denied_phids });
     @denied_user_ids = map { $_->{id} } @$phab_users;
 
-    my %reviewers_hash =  map {$_->name => 1} (@{ $revision->reviewers });
+    my %reviewers_hash =  map {$_->name => 1} @{ $revision->reviewers };
```

Comment:
Missed earlier but also add spaces after opening { brace and before  closing } brace for map().

my %reviewers_hash =  map { $_->name => 1 } @{ $revision->reviewers };

Once Dylans comment is fixed as well this should be ready to merge.